rocAL/rocAL_pybind/example/new_api/README.md

```diff
@@ -1,3 +1,14 @@
+### TESTING ALL THE READER PIPELINES ON PYTHON IN SINGLE SHOT
+
+./TEST_FILE.sh
+
+### TO TEST A SINGLE READER / MULTIPLE READER PIPELINE FROM TEST_FILE.sh
+
+####################################################################################################################################
```

Comment:
please remove this line since it is not needed

Reply:
Resolved